tests/unit/runtime/half_precision/onebit/test_onebit.py

```diff
@@ -1109,27 +1080,21 @@ def test_overflow(self, tmpdir):
 @pytest.mark.parametrize(
     "topo_config",
     [
-        {
-            "num_pp": 1,
-            "num_dp": 4
-        },
         {
             "num_pp": 2,
             "num_dp": 2
         },
-        {
-            "num_pp": 4,
-            "num_dp": 1
-        },
     ],
 )
 class TestOneBitLambFP16Pipeline(DistributedTest):
     world_size = 4
 
     def test(self, topo_config):
+        global_batch_size = 4
+        assert global_batch_size % self.world_size == 0, "Global batch size must be divisible by world size"
```

Comment:
I don't think this is needed since batch-related assertions already done in deepspeed: https://github.com/microsoft/DeepSpeed/blob/ce535945e6dacb6ffbf17a7cbf7543fbd86385dd/deepspeed/runtime/config.py#L878